azerothcore/azerothcore-wotlk
# fix(Scripts/Temple of Ahn Qiraj): Added missing doors.

Fixes #12712



## Issues Addressed:

- Closes #12712

## Tests Performed:

- Tested ingame.

## How to Test the Changes:

Twin Emperors

`.go xyz -8967.704102 1319.128174 -104.250031 531 5`
look to your left, see if door is present and closed. look to your right, see if door is present and open
start encounter
see if both doors are present and closed
`.combatstop`
see if entrance is open
engage again, kill both bosses, see if both doors are now open

Skeram:

`.go xyz -8379.136719 2067.264404 144.814651 531 3.5`
see if door is present and closed
engage boss, kill him
see if door is now open


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
